### PR TITLE
wercker.yml: Add Go AST Scanner (gas)

### PR DESCRIPTION
**Pull request by tonychain**
gas is a static analyzer for security issues:

https://github.com/GoASTScanner/gas

### CODE REVIEW

**Comment by tonychain** [comment]
Note: this branch includes #222, which should get merged first

**Comment by kr** [comment]
:eyes:

File: wercker.yml [hunk 1]
```diff
@@ -41,6 +41,15 @@ build:
           bin/cfmt
           git diff --exit-code
     - script:
+        name: gas
+        code: |
+          go get github.com/GoASTScanner/gas
```
Comment (kr) [inline wercker.yml]: Do we want to pin this to a particular revision of gas? Or maybe put it in the docker image? See the goimports stuff in `docker/wercker/Dockerfile` (and instructions in the Readme there). I can think of tradeoffs on both sides.

**Comment by kr** [comment]
LGTM with one question

File: wercker.yml (hunk 1 above)
Comment (tonychain) [inline wercker.yml]: I think pinning to a particular commit hash of gas makes sense at least, especially as there seem to be pretty frequent (breaking) changes upstream

File: wercker.yml [hunk 2]
```diff
@@ -41,6 +41,18 @@ build:
           bin/cfmt
           git diff --exit-code
     - script:
+        name: gas
+        code: |
+          git clone https://github.com/GoASTScanner/gas $GOPATH/src/github.com/GoASTScanner/gas
+          # gas is somewhat unstable, so pin to a known-good commit hash
+          cd $GOPATH/src/github.com/GoASTScanner/gas && git reset --hard d30c5cde3613e9ba0129febda849e4d4df1d57cd
+          go get github.com/GoASTScanner/gas
```
Comment (kr) [inline wercker.yml]: nit: s/get/install/ to emphasize that we expect this not to do network stuff

**Pull request merged**